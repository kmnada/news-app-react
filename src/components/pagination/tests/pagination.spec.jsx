import React from 'react';
import { shallow } from 'enzyme';
import Pagination from '../pagination';

describe(' Pagination component', () => {
  it('renders default component', () => {
    const wrapper = shallow(<Pagination />);
    expect(wrapper).toMatchSnapshot();
  });
});
