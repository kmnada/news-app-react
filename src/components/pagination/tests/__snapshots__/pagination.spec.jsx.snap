// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Pagination component renders default component 1`] = `ShallowWrapper {}`;
